src/app/design/styles/spacing/index.html

```diff
@@ -1,61 +1,287 @@
+    </div>
+	</div>
+	
+	<div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-subsection-heading" style="background-color: #c1e8fb; width: 50px; height: 25px; margin-bottom: 10px;"></div>
+			<div class="sky-subsection-heading" style="background-color: #c1e8fb; width: 50px; height: 25px;"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>Multiple instances of a type of object need enough vertical spacing so that the objects can be distinguished from each other, but not so much that they seem unrelated. Examples include:</div>
+        <ul>
+          <li>Form inputs</li>
+        </ul>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <div><stache-code>class="sky-margin-stacked-default"</stache-code></div>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+				<div>margin-bottom: 10px</div>
+      </div>
+    </div>
+	</div>
+	
+	<div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-subsection-heading" style="background-color: #c1e8fb; width: 50px; height: 25px; margin-bottom: 20px;"></div>
+			<div class="sky-subsection-heading" style="background-color: #c1e8fb; width: 50px; height: 25px;"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>Sections of loosely-related content need extra spacing to make them easier for the user to discern. Too little spacing will make the sections visually run together.</div>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <div><stache-code>class="sky-margin-stacked-separate"</stache-code></div>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+				<div>margin-bottom: 20px</div>
+      </div>
+    </div>
+	</div>
+</div>
+	
+<div>
+	<div class="sky-subsection-heading">Horizontally-arranged content</div>
+	
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-subsection-heading" style="display: inline-block; background-color: #c1e8fb; width: 25px; height: 50px; margin-right: 5px;"></div>
+			<div class="sky-subsection-heading" style="display: inline-block; background-color: #c1e8fb; width: 25px; height: 50px;"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>Closely-related elements have a small amount of margin between them so that they stay visually linked. Examples include:</div>
+        <ul>
+					<li><a routerLink="/components/status-indicator">Status indicators</a> and text</li>
```

Comment:
it is the label of the status indicator, but I am worried because we have polluted the term "Label" by having a component with that name that competes for usage with status indicators.

Reply:
I think it's still clear to use "label" generically here, but I don't feel strongly either way. However, I still think "Status indicators and text" is unclear because it makes the text sound like it isn't part o the status indicator and because it can also be misinterpreted as meaning space between the status indicator component (including the text/label) and nearby text. If you don't like label, I'd suggest "Status indicator icons and text." And I'd still recommend making "and text" part of the link text.